src/calliope/preprocess/model_math.py

```diff
@@ -1,55 +1,56 @@
 """Calliope math handling with interfaces for pre-defined and user-defined files."""
 
+import importlib.resources
 import logging
 from copy import deepcopy
-from importlib.resources import files
 from pathlib import Path
 
-from calliope import exceptions
 from calliope.attrdict import AttrDict
+from calliope.exceptions import ModelError
 from calliope.util.schema import MATH_SCHEMA, validate_dict
 from calliope.util.tools import relative_path
 
 LOGGER = logging.getLogger(__name__)
-MATH_DIR = files("calliope") / "math"
 
 
-class ModelMath:
-    """Calliope math preprocessing."""
+class CalliopeMath:
+    """Calliope math handling."""
 
     ATTRS_TO_SAVE = ("history", "data")
+    ATTRS_TO_LOAD = ("history",)
```

Comment:
Why are we not loading back the math data?
I was doing it because the file structure might have changed, meaning the "rebuild" would fail since the reference directories would be different...